src/app/cube/details/version-switch/version-switch.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="inner">
+  <div *ngIf="hasRevisions === true; then revisions; else noRevisions"></div>
```

Comment:
hasRevisions is a boolean. You can check if it is true with `*ngIf="hasRevisions"`.

Reply:
Resolved with Remove === true.